src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1508,6 +1508,13 @@ static CORINFO_RESOLVED_TOKEN CreateResolvedTokenFromMethod(CorInfoImpl jitInter
             return result != null ? ObjectToHandle(result) : null;
         }
 
+        private CORINFO_METHOD_STRUCT_* getInstantiatedEntry(CORINFO_METHOD_STRUCT_* ftn, CORINFO_METHOD_STRUCT_** methodArg, CORINFO_CLASS_STRUCT_** classArg)
+        {
+            *methodArg = null;
+            *classArg = null;
+            return null;
```

Comment:
worth leaving a note why it's no-op for NAOT/R2R?

Reply:
Yes, good point.

Also this should not be too hard to implement, but I don't know how to do it, and it won't get used. Currently R2R lacks the devirtualization support, and NAOT doesn't handle arrays the same way.